addon/types/model_export/model.py

```diff
@@ -1,6 +1,7 @@
 import bpy
 import shutil
 import subprocess
+from math import degrees
```

Comment:
I'd probably import math itself here, though I don't have any valid reason for that.

Reply:
I've left this line as-is for now, because I don't know if it uses more resources to `import math` rather than just what's being used; `degrees()`.

Reply:
I don't think it does, but it doesn't really matter; keep it as-is, I was just being weird.